tools/nni_cmd/tests/mock/restful_server.py

```diff
@@ -0,0 +1,183 @@
+    responses.add(
+        responses.GET,
+        "http://localhost:8080/api/v1/nni/experiment",
+        json={"id":"bkfhOdUl","revision":5,"execDuration":10,"logDir":"/home/shinyang/nni-experiments/bkfhOdUl",
+        "nextSequenceId":2,"params":{"authorName":"default","experimentName":"example_sklearn-classification",
+        "trialConcurrency":1,"maxExecDuration":3600,"maxTrialNum":1,
+        "searchSpace":"{\"C\": {\"_type\": \"uniform\", \"_value\": [0.1, 1]}, \
+        \"kernel\": {\"_type\": \"choice\", \"_value\": [\"linear\", \"rbf\", \"poly\", \"sigmoid\"]}, \
+        \"degree\": {\"_type\": \"choice\", \"_value\": [1, 2, 3, 4]}, \"gamma\": {\"_type\": \"uniform\", \
+        \"_value\": [0.01, 0.1]}, \"coef0 \": {\"_type\": \"uniform\", \"_value\": [0.01, 0.1]}}",
+        "trainingServicePlatform":"local","tuner":{"builtinTunerName":"TPE","classArgs":{"optimize_mode":"maximize"}, \
+        "checkpointDir":"/home/shinyang/nni-experiments/bkfhOdUl/checkpoint"},"versionCheck":"true", \
+        "clusterMetaData":[{"key":"codeDir","value":"/home/shinyang/folder/examples/trials/sklearn/classification/."}, \
+        {"key":"command","value":"python3 main.py"}]},"startTime":1600326895536,"endTime":1600326910605},
+        status=200
+    )
+ 
+def mock_update_experiment_profile():
+    responses.add(
+        responses.PUT, 'http://localhost:8080/api/v1/nni/experiment',
+        json={"status":"RUNNING","errors":[]},
+        status=200,
+        content_type='application/json',
+    )
+
+def mock_import_data():
+    responses.add(
+        responses.POST, 'http://localhost:8080/api/v1/nni/experiment/import-data',
+        json={"result":"data"},
+        status=200,
+        content_type='application/json',
+    )
+
+def mock_start_experiment():
+    responses.add(
+        responses.POST, 'http://localhost:8080/api/v1/nni/experiment',
+        json={"status":"RUNNING","errors":[]},
+        status=200,
+        content_type='application/json',
+    )
+
+def mock_get_trial_job_statistics():
+    responses.add(
+        responses.GET, 'http://localhost:8080/api/v1/nni/job-statistics',
+        json=[{"trialJobStatus":"SUCCEEDED","trialJobNumber":1}],
+        status=200,
+        content_type='application/json',
+    )
+
+def mock_set_cluster_metadata():
+    responses.add(
+        responses.PUT, 'http://localhost:8080/api/v1/nni/experiment/cluster-metadata',
+        json=[{"trialJobStatus":"SUCCEEDED","trialJobNumber":1}],
+        status=200,
+        content_type='application/json',
+    )
+
+def mock_list_trial_jobs():
+    responses.add(
+        responses.GET, 'http://localhost:8080/api/v1/nni/trial-jobs',
+        json=[{"id":"GPInz","status":"SUCCEEDED","hyperParameters":["{\"parameter_id\":0, \
+        \"parameter_source\":\"algorithm\",\"parameters\":{\"C\":0.8748364659110364, \
+        \"kernel\":\"linear\",\"degree\":1,\"gamma\":0.040451413392113666,\"coef0 \":0.040353532973662266}, \
```

Comment:
I remember the extra space in `coef0 ` has been removed in another PR?

Reply:
updated, in face they are fake data, does not need to make sure the content in latest format, as long as we could get the required response from this request.